doc/script_commands.txt

```diff
@@ -5389,7 +5389,7 @@ and theirs val1, val2, val3, and val4 usage in source.
 perform a complete removal of all statuses (although permanent ones will re-apply).
 
 'sc_end_class' works like 'sc_end' but will remove all status effects from any learned
-skill on the invoking character.
+skill on the invoking character. if <job_id> was given it will end the effect for that job.
```

Comment:
`If <job_id> is provided it will end the effect for the that job.`

Reply:
Sorry. I just realized I made a typo myself. I corrected the comment above to what it should be. Also, don't forget this: https://github.com/rathena/rathena/pull/2673#discussion_r154879679